web-analogs.md

```diff
@@ -157,9 +157,8 @@ var container = new Container( // grey box
 ### Setting Container Width
 
 To specify the width of a [Container](https://docs.flutter.io/flutter/widgets/Container-class.html)
-widget you set its ```width``` property. This is a fixed width, unlike the
-CSS max-width property which adjusts container width up to a maximum value. To
-mimic that effect in Flutter, use the ```constraints``` property of the Container.
+widget you set its ```width``` property just like in CSS. To
+mimic CSS's ```max-width``` in Flutter, use the ```constraints``` property of the Container.
```

Comment:
Oy. Suddenly this page switches from one backtick for code to 3. This is incorrect. It should be one. But you don't have to fix it.

Reply:
"just like in CSS" => as you would in CSS.
delete "in Flutter"
Please replace all ``` with `.